### PR TITLE
For #1749 - Land ETP Strings

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -718,4 +718,56 @@
...
+
+    <!-- Enhanced Tracking Protection -->
+    <!-- Link displayed in enhanced tracking protection panel to access tracking protection settings -->
+    <string name="etp_settings">Protection Settings</string>
+    <!-- Preference title for enhanced tracking protection settings -->
+    <string name="preference_enhanced_tracking_protection">Enhanced Tracking Protection</string>
+    <!-- Title for the description of enhanced tracking protection -->
+    <string name="preference_enhanced_tracking_protection_explanation_title">Browse without being followed</string>
+    <!-- Description of enhanced tracking protection -->
+    <string name="preference_enhanced_tracking_protection_explanation">Keep your data to yourself. Firefox protects you from many of the most common trackers that follow what you do online.</string>
```
Comment (boek) [inline app/src/main/res/values/strings.xml]: Should `Firefox` be the app_name?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -718,4 +718,56 @@
...
+    <!-- Category of trackers (fingerprinters) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_fingerprinters_title">Fingerprinters</string>
+    <!-- Description of fingerprinters that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_fingerprinters_description">Stops uniquely identifiable data from being collected about your device that can be used for tracking purposes.</string>
+    <!-- Category of trackers (tracking content) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_tracking_content_title">Tracking Content</string>
+    <!-- Description of tracking content that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_tracking_content_description">Stops outside ads, videos, and other content from loading that contains tracking code. May affect some website functionality.</string>
+    <!-- Enhanced Tracking Protection Onboarding Message shown in a dialog above the toolbar -->
+    <string name="etp_onboarding_message">When the shield changes colors, Firefox is blocking trackers on a website. Tap it open to see what’s being blocked.</string>
```
Comment (boek) [inline app/src/main/res/values/strings.xml]: Firefox/app_name?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ekager) [reply app/src/main/res/values/strings.xml]: @mheubusch could we get a confirmation that this string should say "Firefox" and not be translated?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ekager) [reply app/src/main/res/values/strings.xml]: @mheubusch could we get a confirmation that this string should say "Firefox" and not be translated?

Comment (mheubusch) [reply app/src/main/res/values/strings.xml]: @ekager The string should have the parameter for the app short name (Firefox Preview in this case) and only that name should not be localized. All other portions of the string should be localized.

**Comment by ekager** [comment]
Good catch @boek ! Got confirmation that they should take params, updating now

**Pull request merged**